docs/Hardening.md

```diff
@@ -0,0 +1,197 @@
+# Linux Hardening while using OMS Log Collector
+We support some industry standard hardening (please see [Supported Hardening or Customization](#supported-hardening-or-customization) for more details).
+We also can provide some [troubleshooting steps](#common-hardening-issues) for customized hardening.
+
+# Table of Contents
+- [Supported Hardening or Customization](#supported-hardening-or-customization)
+- [Common Hardening Issues](#common-hardening-issues)
+- [Documentation](#documentation)
+
+## Supported Hardening or Customization
+### Hardening with FIPS
+FIPS hardening is supported in OMS versions 1.13 and newer.
+
+How to check if FIPS is enabled:
+```
+$> cat /proc/sys/crypto/fips_enabled
+1
+$> sysctl crypto.fips_enabled
+crypto.fips_enabled = 1
+```
+
+### Hardening with Cylance Anti-Virus
+Cylance Anti-Virus hardening is supported in OMS versions 1.12 and newer.
+
+### Not Yet Supported Hardening
+- SELinux
+- Center for Internet Security (CIS)
+
+## Common Hardening Issues
+If you're using a customized hardening setup, and are having issues running the OMS Agent, try going through these common problems encountered with hardening.
+### Directory Permissions
+#### Issue
+Several major folders in Linux need to have appropriate file permissions, otherwise the OMS Agent won't run properly. Specifically, we need the following:
+| Directory | Permissions (numeric value) | Permissions (symbolic notation) |
+| --- | --- | --- |
```

Comment:
how about `/opt/`?

Reply:
Abder didn't mention it when listing out which directories are required, but that seems important - @abenbachir are the permissions required the same as the majority of the others (775)?

Reply:
Added `/opt` with 775 just in case:
https://github.com/microsoft/OMS-Agent-for-Linux/pull/1269/commits/b6a96c6bef0fd670a3c6676c17026d5228135af5#diff-ff8800189743d6567bcda8a83de5b22c943b28c51e4488cce0ea2b185716cc37R38